README.md

```diff
@@ -21,44 +21,55 @@ This action checks dependency licence issues using [ort](https://github.com/oss-
 See [action.yml](action.yml)
 
 ### Own repository
+
 ```yaml
 on:
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+    # Allows you to run this workflow manually from the Actions tab
+    workflow_dispatch:
 
 jobs:
-  tortellini:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - uses: tortellini-tools/action@main
-      with:
-        curations: .tortellini.yml
-        rules: https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/rules.kts
-        classifications: https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/license-classifications.yml
+    tortellini:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: tortellini-tools/action@main
+            - uses: actions/upload-artifact@v2
+              with:
+                  name: tortellini-result
+                  path: .tortellini/out
 ```
 
+Tortellini action will generate `.tortellini/out/scan-report-web-app.html` file.
+The HTML file can be downloaded from the [workflow page](https://github.com/actions/upload-artifact#where-does-the-upload-go) by using the GitHubs upload-artifact action.
```

Comment:
This sentence is a bit confusing to me. I would not expect a pointer to "the workflow page" to point to the documentation of the action.

Maybe something like:

The HTML file can be downloaded from the workflow summary page (see the documentation of the [`upload-artifact` Github Action](https://github.com/actions/upload-artifact#where-does-the-upload-go).